src/category.rs

```diff
@@ -398,4 +424,30 @@ mod tests {
         let expected = vec![("Cat 3".to_string(), 6), ("Cat 1".to_string(), 3)];
         assert_eq!(expected, categories);
     }
+
+    #[test]
+    fn category_parent_categories_includes_path_to_node_with_count() {
+        let conn = pg_connection();
+        conn.batch_execute(
```

Comment:
Can we use Diesel's public API for this (I need to update the other tests). You can write this as

```
insert_into(categories)
    .values(vec![
        (category.eq("Cat 1"), slug.eq("cat1"), crates_cnt.eq(1)),
        ...
    ])
```

Reply:
Done this (and fixed up the other tests too).